src/features/chat/Chats.js

```diff
@@ -12,12 +16,50 @@ const StyledChats = styled.div`
   background-color: black;
 `;
 
+const BASE_URL = process.env.REACT_APP_AXIOS_BASE_URL;
+
 function Chats() {
```

Comment:
잠깐 저희가 언급했었던거  처럼 s 를 붙이는것 보다는 list 라는 네이밍으로가는건 어떨까요?
```suggestion
function ChatList() {
```

Reply:
정리 완료!